system-tests/__snapshots__/results_spec.ts.js

```diff
@@ -14,6 +14,7 @@ exports['module api and after:run results'] = `
     "arch": "x64",
     "baseUrl": null,
     "blockHosts": null,
+    "experimentalBurnIn": false,
```

Comment:
nit: Is it posssible to alphabetize this? I feel like Chris might have said it's not automatic. 😅 

Regardless, approval given for this change as the codeowner of this file.

Reply:
I don't think we can but I cant remember. Either way this should not be there, so I am going to revert the commit it was introduced which was bb5046c91e023930341ad2fcc2372ff18573dec6

Reply:
reverted in 83842a6f82